docs/source/_json/breadcrumbs.json

```diff
@@ -6,14 +6,12 @@ content-type: application/json
 
 [
   {
-    "data": {
-      "items": [
-        {
-          "title": "Welcome to Plone", 
-          "url": "http://localhost:55001/plone/front-page"
-        }
-      ]
-    }, 
-    "id": "breadcrumbs"
+    "@id": "breadcrumbs", 
```

Comment:
@ebrehault if we use "@id" the value needs to contain the full URL of the object. Otherwise we have to stick with "id".